### ISSUE TITLE
Seatmap rendering crashes javascript if the seatmap background does not exist.

### ISSUE THREAD

**Issue by halvors** [InfectedLan/InfectedAPI#168]
Seatmap rendering crashes javascript if the seatmap background does not exist.

**Comment by petterroea** [comment]
It doesn't crash javascript as far as i know, as we simply set the style to background-image:url("image");. Are you sure this is the cause?

**Comment by halvors** [comment]
Yes. Could you try debugging this? I'm pretty sure it is when there is no background, or could it be that there is no rows or seats maybe? Anyway, it should handle it.

**Comment by petterroea** [comment]
No rows or seats propabily. What seatmap and what server could i use as a testcase?

**Comment by halvors** [comment]
Use the test.infected.no server, that has a seatmap. Just pick one. Example is the earlier events 1 or something that was adapted from the old ticket system and does not have a rows and seats.
